azure-pipelines.yml

```diff
@@ -79,18 +81,30 @@ stages:
             Write-Host "##vso[task.setvariable variable=SC_VERSION;isOutput=true]$version"
 
             Rename-Item -Path "$env:AGENT_BUILDDIRECTORY/a/sonar-scanner-vsts-0.0.2-SNAPSHOT-sonarcloud.vsix" -NewName $extensionNewName
+
+            $version = Get-AzureDevopsTaskVersion "$env:BUILD_SOURCESDIRECTORY/build/extensions" "sonarcloud" "analyze" "v1"
+            Write-Host "##vso[task.setvariable variable=SC_V1_ANALYZE_VERSION;isOutput=true]$version"
+
+            $version = Get-AzureDevopsTaskVersion "$env:BUILD_SOURCESDIRECTORY/build/extensions" "sonarcloud" "prepare" "v1"
+            Write-Host "##vso[task.setvariable variable=SC_V1_PREPARE_VERSION;isOutput=true]$version"
+
+            $version = Get-AzureDevopsTaskVersion "$env:BUILD_SOURCESDIRECTORY/build/extensions" "sonarcloud" "publish" "v1"
+            Write-Host "##vso[task.setvariable variable=SC_V1_PUBLISH_VERSION;isOutput=true]$version"
         pwsh: true
     - task: PublishBuildArtifacts@1
       displayName: 'Publish artifacts on the build'
       inputs:
         PathtoPublish: '$(Agent.BuildDirectory)/a'
         ArtifactName: 'extensions'
 - stage: unpublish
+  condition: and(succeeded(), ne(variables['Build.SourceBranch'], 'refs/heads/master'))
```

Comment:
1) Why not run the tests on master too?

2) the "Build.SourceBranch" condition is repeated three times in the yaml. It would be easier to understand and maintain if it the value was asigned to a variable with more friendly name e.g. `runITs`

Reply:
I thought the whole process is not necessary on master, but we can run it for sure, i'll remove the condition.

Reply:
Condition removed